packages/forms/src/directives/validators.ts

```diff
@@ -167,14 +167,14 @@ export const MAX_VALIDATOR: StaticProvider = {
   selector:
       'input[type=number][max][formControlName],input[type=number][max][formControl],input[type=number][max][ngModel]',
   providers: [MAX_VALIDATOR],
-  host: {'[attr.max]': 'max ? max : null'}
+  host: {'[attr.max]': 'max'}
```

Comment:
I think this might be a good place for nullish coalescing (`??`), since this is one of its main use-cases: if there is a `null` or `undefined` - user default value (`null`), otherwise - use provided value.
```suggestion
  host: {'[attr.max]': 'max ?? null'}
```
The only value that we are going to cover with this would be `undefined` though, in all other cases `max` and `max ?? null` would be equivalent.

Reply:
yes sounds good